src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -697,16 +697,13 @@ public function from($collection, $as = null)
 
     /**
      * @inheritdoc
+     * @return boolean
      */
-    public function truncate()
+    public function truncate(): bool
```

Comment:
if you added type hints, remove doc blocks please. 
We want to use common style in php classes, so i suggest to delete return type hints and add type hints in refactoring task in future